tasks/debian_repositories.yml

```diff
@@ -9,4 +9,5 @@
       Package: *
       Pin: release a=unstable
       Pin-Priority: 150
-
+- apt:
+    update_cache: true
```

Comment:
This seems to break the idempotence test on debian8 https://travis-ci.org/adamruzicka/ansible-wireguard/jobs/495073226#L556